ecl/eclagent/eclagent.cpp

```diff
@@ -1608,18 +1608,26 @@ void EclAgent::selectCluster(const char *newCluster)
         if (!streq(oldCluster, newCluster))
             throw MakeStringException(-1, "Error - cannot switch cluster in hthor jobs");
     }
+    if(!streq(oldCluster, newCluster))
```

Comment:
trivial/formatting: missing space after 'if' for consistency.

Reply:
minor: to simply the code slightly, and avoid the double streq (if equal), you could move the HThorCluster test inside the !streq block, i.e.:
```
    if (!streq(oldCluster, newCluster))
    {
        // If the current cluster is an hthor cluster, it's an error to change it...
        if (getClusterType(clusterType)==HThorCluster)
            throw MakeStringException(-1, "Error - cannot switch cluster in hthor jobs");

        WorkunitUpdate wu = updateWorkUnit();
        wu->setClusterName(newCluster);
        clusterWidth = -1;
    }
```